### PR TITLE
Add Rocky Linux 8.3 x86_64

### PR DESCRIPTION
**Pull request by sklimchuk**
Signed-off-by: Sergey Klimchuk <sergey.klimchuk@gmail.com>

## Description
Inspired by https://rockylinux.org, let's support their intentions.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: packer_templates/rocky/http/8/ks.cfg [hunk 1]
```diff
@@ -0,0 +1,77 @@
+install
```
Comment (tas50) [inline packer_templates/rocky/http/8/ks.cfg:1]: Is there any reason not to just use the centos 8 ks file here so we avoid the need to manage another file?

File: packer_templates/rocky/scripts/cleanup.sh [hunk 2]
```diff
@@ -0,0 +1,113 @@
+#!/bin/sh -eux
```
Comment (tas50) [inline packer_templates/rocky/scripts/cleanup.sh:1]: If these aren't specific to Rocky they should just source the centos files like we do with Alma Linux

File: packer_templates/rocky/http/8/ks.cfg (hunk 1 above)
Comment (sklimchuk) [reply packer_templates/rocky/http/8/ks.cfg:1]: Actually it's the same, left here just to have all files in place w/o additional actions.

File: packer_templates/rocky/scripts/cleanup.sh (hunk 2 above)
Comment (sklimchuk) [reply packer_templates/rocky/scripts/cleanup.sh:1]: At the moment these scripts have zero specific to Rocky actually, but who knows maybe sometime they will be extended. Added there as they used in "scripts" section of main JSON file.

Comment (sklimchuk) [reply packer_templates/rocky/scripts/cleanup.sh:1]: Actually I can rewrite these includes like you did for springdalelinux.

**Comment by sklimchuk** [comment]
Closing this PR if favor of new one.

**Pull request closed without merging**